plugin/src/main/groovy/com/here/gradle/plugins/jobdsl/RestJobManagement.groovy

```diff
@@ -125,6 +125,10 @@ class RestJobManagement extends AbstractJobManagement implements DeferredJobMana
         jenkinsJobManagement = new JenkinsJobManagement(System.out, [:], workspace)
 
         restClient = new RESTClient(jenkinsUrl)
+        if (proxyUrl != null && !proxyUrl.isEmpty()) {
+            URI proxyURI = new URI(proxyUrl)
+            restClient.setProxy(proxyURI.host, proxyURI.port, proxyURI.scheme)
+        }
```

Comment:
Please put this block below `restClient.handler.failure = { it }` for better readability.